AOBS-491: Fix to support TIMESTAMP_TZ and TIMESTAMP_LTZ in Parquet file (#11)
outrauk/dataiku-plugin-snowflake-hdfs: README.md

## Building in PyCharm

> Make sure that wget is installed. For Mac run `brew install wget` to install wget.

Reviewer: ```suggestion
Make sure that [wget](https://www.gnu.org/software/wget/) is installed. For macOS, you can install via `brew install wget`.
```

1. Make sure you can build the project by doing all of the `pip` steps above
2. In PyCharm, run the _Unit Tests_ configuration.

> ## Known Issues

Reviewer: I think you're conflating two known issues:

1. Snowflake cannot write timezone'd times to Parquet (AOBS-491; this PR). The README should explain the consequence of our work around (that timezone information is dropped, potentially link to SF articles about it)
2. Dataiku can only read date times as `int96` rather than the more standard annotated Logical Types. You can simply say `Dataiku Parquet issues documented in #12`, or make no mention of it as its outside the scope of this PR.

1. Make sure you can build the project by doing all of the `pip` steps above
2. In PyCharm, run the _Unit Tests_ configuration.

## Known Issues

> + Currently Parquet does not support TIMESTAMP_TZ and TIMESTAMP_LTZ data types so they are converted to TIMESTAMP_NTZ.

Reviewer: ```suggestion
* Neither Snowflake nor Dataiku use [Logical Type annotations](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#datetime-types) for timezone offsets (Dataiku doesn't use Logical Types at all). When synching from Snowflake to HDFS, the plugin casts any `TIMESTAMP_TZ` or `TIMESTAMP_LTZ` columns to `TIMESTAMP_NTZ` which simply drops the timezone offset attribute. For greater control of this behaviour, transform your Snowflake table before passing it to this plugin. Consider using `CONVERT_TIMEZONE('UTC', t."date")::TIMESTAMP_NTZ`.
```

>   As a result timezone details are removed from the TIMESTAMP. See [Date & Time Data Types in Snowflake](https://docs.snowflake.com/en/sql-reference/data-types-datetime.html) for more details.

Reviewer: ```suggestion
  See [Date & Time Data Types in Snowflake](https://docs.snowflake.com/en/sql-reference/data-types-datetime.html) for more details.
```

> + All TIMESTAMP (LTZ, TZ, NTZ) and DATE columns from Snowflake are stored as INTEGER in Parquet.

Reviewer: ```suggestion
* Snowflake stores all `TIMESTAMP_NTZ` and `DATE` columns as annotated logical types in Parquet. Because Dataiku does not support logical types, these appear in Dataiku as `bigint` (`int64` as `datetime`) and `int` (`int32` as `date`). More details and workarounds are described in #12 
```
